### ISSUE TITLE
HTML5 exported project does not load in iOS 14 Safari

### ISSUE THREAD

**Issue by thinwire** [godotengine/godot#43473]
**Godot version: 3.2.2, 3.2.3, 3.2.4 beta 1**

**OS/device including version: iOS 14.0.1, iOS 14.1, iOS 14.2 / iPad Pro, iPad Mini 4**
Problem manifests on unknown number of iOS 14 devices; reported to work on iPhones running iOS 14.
Works fine on iOS 12 devices of similar or lower spec. Works on iOS 14.1 simulator devices.

**Issue description:**
Godot games exported through HTML5 export fail to start up on some iOS 14 devices. Godot loader completes preload, page crashes during game initialization, then automatically retries and shows an error screen.
This seems to be restricted to some iOS 14 devices, at least the iPad Mini 4 and iPad Pro do not work.
I have been unable to get a remote debugging connection up to the test device I was provided, so can unfortunately not provide a browser crash trace.

**Steps to reproduce:**
Create a project with GLES2 renderer, create a 2D scene with anything in it, export result as HTML5. Game should start right up on any device except some iDevices running iOS 14.

**Minimal reproduction project:**
Minimal project provided. [GodotTest.zip](https://github.com/godotengine/godot/files/5530122/GodotTest.zip)

**Comment by akien-mga** [comment]
Ah Safari <3

**Comment by cesarizu** [comment]
On my project, I fixed this problem by unchecking the `window/size/fullscreen` checkbox on the project settings.

**Comment by Calinou** [comment]
@cesarizu Good catch, but @thinwire's minimal reproduction project doesn't have fullscreen enabled.

**Comment by cesarizu** [comment]
@Calinou that's true, I missed that bit from the test project.

**Comment by gianmichelemariani** [comment]
This is working for me now re-exporting the minimal project with 3.2.4 beta5.
I have to say performance on Macs is a lot better with each beta. With iOS, even with a simple scene and a sprite moving back and forth, is really choppy when in full screen. A smaller window does help a lot in this regard.

**Comment by Calinou** [comment]
> With iOS, even with a simple scene and a sprite moving back and forth, is really choppy when in full screen. A smaller window does help a lot in this regard.

On mobile, you often want to use a [reduced viewport resolution](https://docs.godotengine.org/en/latest/tutorials/rendering/multiple_resolutions.html) (1280×720 or lower), especially in 3D.

**Comment by gianmichelemariani** [comment]
Yeah I am using a smaller resolution, and everything is smooth when rendered at that resolution in html5.
If I go full screen though (with black bars around) it gets real choppy.
I'm sure it will get there. The progress so far for the web platform has been super steady!

**Comment by chenriquevz** [comment]
The problem persists using ipad mini 4 iOS 15.8.2 (the latest), using godot 3.5.3 (latest) or godot 3.6 RC1 (latest). An empty project will not work as described "Godot loader completes preload, page crashes during game initialization, then automatically retries and shows an error screen.".

I have access to an ipad mini 4 if there is need for any troubleshooting. I managed to grab a crash analytics from the device.
https://gist.github.com/chenriquevz/f1c943d4082ab31d2e0b16381618cd41
